Wrappers/Python/ccpi/viewer/widgets/slider.py

```diff
@@ -0,0 +1,133 @@
+import vtk
+
+class SliderProperties:
+    def __init__(self):
+        self.tube_width = 0.004
+        self.slider_length = 0.025
+        self.slider_width = 0.015
+        self.end_cap_length = 0.008
+        self.end_cap_width = 0.02
+        self.title_height = 0.02
+        self.label_height = 0.02
+
+        self.value_minimum = 0.0
+        self.value_maximum = 1.0
+        self.value_initial = 1.0
+
+        self.orientation = 'horizontal'
+        self.offset = 0.12
+
+        self.p1 = [self.offset, self.end_cap_width * 1.1]
+        self.p2 = [1 - self.offset, self.end_cap_width * 1.1]
+
+        self.title = None
+
+        self.title_color = 'Black'
+        self.label_color = 'White'
+        self.value_color = 'White'
+        self.value_background_color = 'Black'
+        self.slider_color = 'Lime'
+        self.selected_color = 'Lime'
+        self.bar_color = 'Gray'
+        self.bar_ends_color = 'Yellow'
+
+
+    def get_slider_widget(self, orientation='horizontal', offset=0.12):
```

Comment:
I understand this method returns the widget but more importantly it creates it. Can we change the name to "Make" or "Create" or similar?